code/scripts/setup/environment/superscript/superscript.sh

```diff
@@ -0,0 +1,166 @@
+#!bin/bash
+green=`tput setaf 2` 
```

Comment:
Use $(...) notation instead of legacy backticks `...`. Please do that across the whole script.